tests/functional/test_ems_end2end.py

```diff
@@ -219,23 +219,35 @@ def setup_test_processes(cls):
         cls.PROCESS_FLOOD_DETECTION_ID = "FloodDetection"
         cls.PROCESS_ICE_DAYS_ID = "Finch_IceDays"
         cls.PROCESS_SUBSET_BBOX_ID = "ColibriFlyingpigeon_SubsetBbox"
+        cls.PROCESS_SUBSET_ESGF = "SubsetESGF"
+        cls.PROCESS_SUBSET_NASAESGF = "SubsetNASAESGF"
         cls.PROCESS_WORKFLOW_ID = "Workflow"
         cls.PROCESS_WORKFLOW_SC_ID = "WorkflowSimpleChain"
         cls.PROCESS_WORKFLOW_S2P_ID = "WorkflowS2ProbaV"
         cls.PROCESS_WORKFLOW_CUSTOM_ID = "CustomWorkflow"
         cls.PROCESS_WORKFLOW_FLOOD_DETECTION_ID = "WorkflowFloodDetection"
         cls.PROCESS_WORKFLOW_SUBSET_ICE_DAYS = "WorkflowSubsetIceDays"
+        cls.PROCESS_WORKFLOW_SubsetLLNL_SubsetCRIM = "WorkflowSubsetLLNL_SubsetCRIM"
+        cls.PROCESS_WORKFLOW_SubsetNASAESGF_SubsetCRIM = "WorkflowSubsetNASAESGF_SubsetCRIM"
```

Comment:
make all caps to remain consistent in var naming, use underscore to split acronyms as necessary